test/partiql-tests-runner/src/test/kotlin/org/partiql/runner/Util.kt

```diff
@@ -7,17 +7,16 @@ import com.amazon.ion.IonSexp
 import com.amazon.ion.IonStruct
 import com.amazon.ion.IonSystem
 import com.amazon.ion.IonValue
+import org.partiql.lang.eval.BAG_ANNOTATION
 import org.partiql.lang.eval.ExprValue
 import org.partiql.lang.eval.ExprValueFactory
 import org.partiql.lang.eval.ExprValueType
+import org.partiql.lang.eval.MISSING_ANNOTATION
 import org.partiql.lang.eval.StructOrdering
 import org.partiql.lang.eval.name
 import org.partiql.lang.eval.namedValue
 import org.partiql.lang.eval.stringValue
 
-const val BAG_ANNOTATION = "\$bag"
-const val MISSING_ANNOTATION = "\$missing"
-
 /**
```

Comment:
Could we actually delete these extension functions and use the default `ExprValue` to `IonValue` conversion function? These extension functions only existed previously because the annotations differed between the conformance tests and the default Kotlin tests using Ion to represent PartiQL values.

Reply:
Yeah, I will do this in the PR to remove `ionValue` property in `ExprValue` interface

Reply:
Ok, can you cut an issue to track?

Reply:
https://github.com/partiql/partiql-lang-kotlin/issues/875